hooks/pnpmfile/test/index.ts

```diff
@@ -44,19 +44,19 @@ test('filterLog hook combines with the global hook', () => {
   })).toBeFalsy()
 })
 
-test('pnpmfileChecksum resolves to undefined when pnpmfile does not exist', async () => {
+test('calculatePnpmfileChecksum is undefined when pnpmfile does not exist', async () => {
   const hooks = requireHooks(__dirname, { pnpmfile: 'file-that-does-not-exist.js' })
-  expect(await hooks.pnpmfileChecksum).toBeUndefined()
+  expect(hooks.calculatePnpmfileChecksum).toBeUndefined()
 })
 
-test('pnpmfileChecksum resolves to hash string for existing pnpmfile', async () => {
+test('calculatePnpmfileChecksum resolves to hash string for existing pnpmfile', async () => {
   const pnpmfile = path.join(__dirname, 'pnpmfiles/readPackageNoObject.js')
   const hooks = requireHooks(__dirname, { pnpmfile })
-  expect(typeof await hooks.pnpmfileChecksum).toBe('string')
+  expect(typeof await hooks.calculatePnpmfileChecksum?.()).toBe('string')
 })
 
-test('pnpmfileChecksum resolves to hash string for existing pnpmfile even when it exports undefined', async () => {
+test('calculatePnpmfileChecksum is undefined if pnpmfile even when it exports undefined', async () => {
   const pnpmfile = path.join(__dirname, 'pnpmfiles/undefined.js')
   const hooks = requireHooks(__dirname, { pnpmfile })
-  expect(typeof await hooks.pnpmfileChecksum).toBe('string')
+  expect(hooks.calculatePnpmfileChecksum).toBeUndefined()
```

Comment:
@zkochan So if the user has a pnpmfile that exports `undefined`, `pnpmfileChecksum` won't be added to the lockfile.

Reply:
Correct. It doesn't do anything, so it is like if there was not pnpmfile at all.